components/Comment/CommentAnonymously.tsx

```diff
@@ -0,0 +1,29 @@
+import { ReactElement } from "react";
+import Switch from "@mui/material/Switch";
+import { css, StyleSheet } from "aphrodite";
+
+type Props = {
+  id: string;
```

Comment:
We have a root type called ID! :) 
```suggestion
  id: ID;
```

Reply:
Adjusted